### PR TITLE
OSDOCS-7087: remove "optional" from greenboot

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-7087

Link to docs preview:
https://63035--docspreview.netlify.app/microshift/latest/microshift_install/microshift-greenboot.html (intro text)
https://63035--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#installing-microshift-from-rpm-package_microshift-install-rpm (procedure)
https://63035--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree.html#adding-microshift-service-to-blueprint_microshift-embed-in-rpm-ostree (table footnote)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: microshift_install/microshift-greenboot.adoc [hunk 1]
```diff
@@ -6,13 +6,13 @@ include::_attributes/attributes-microshift.adoc[]
...
 
 Using greenboot can reduce your risk of being locked out of edge devices during updates and prevent a significant interruption of service if an update fails. When a failure is detected, the system boots into the last known working configuration using the `rpm-ostree` rollback capability.
 
-A {product-title} health check script is included in the `microshift-greenboot` RPM. The `greenboot-default-health-check` RPM includes health check scripts verifying that DNS and `ostree` services are accessible. You can also create your own health check scripts based on the workloads you are running. You can write one that verifies that an application has started, for example.
+A {product-title} application health check script is included in the `microshift-greenboot` RPM. The `greenboot-default-health-check` RPM also includes health check scripts verifying that DNS and `ostree` services are accessible. In addition, you can create your own health check scripts for the workloads you are running. You can write one that verifies that an application has started, for example.
```
Comment (ggiguash) [inline microshift_install/microshift-greenboot.adoc:15]:
```suggestion
A {product-title} application health check script is included in the `microshift-greenboot` RPM. The `greenboot-default-health-checks` RPM also includes health check scripts verifying that DNS and `ostree` services are accessible. In addition, you can create your own health check scripts for the workloads you are running. You can write one that verifies that an application has started, for example.
```

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 2]
```diff
@@ -37,7 +37,7 @@ enabled = ["microshift"]
 EOF
 ----
 [.small]
-<1> Optional `microshift-greenboot` RPM. For more information, read the "Greenboot health check" guide in the "Running Applications" section.
+<1> The `microshift-greenboot` RPM. For more information, read the "Greenboot health check" guide in the "Running Applications" section.
```
Comment (ggiguash) [inline modules/microshift-adding-service-to-blueprint.adoc:40]: I think lines 31-33 should be removed together with the note. When `microshift` RPM is installed, it pulls greenboot automatically, alongside other RPM dependencies.

File: microshift_install/microshift-greenboot.adoc [hunk 3]
```diff
@@ -6,13 +6,13 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Greenboot is the generic health check framework for the `systemd` service on RPM-OSTree-based systems. The `microshift-greenboot` RPM and `greenboot-default-health-check` are optional RPM packages you can install. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble.
+Greenboot is the generic health check framework for the `systemd` service on RPM-OSTree-based systems. The `microshift-greenboot` RPM and `greenboot-default-health-check` are RPM packages installed with {product-title}. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_install/microshift-greenboot.adoc:9]: The correction of the RPM is covered in the doc bug PR.

File: microshift_install/microshift-greenboot.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply microshift_install/microshift-greenboot.adoc:15]: Same here, in the bug PR.

File: modules/microshift-adding-service-to-blueprint.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/microshift-adding-service-to-blueprint.adoc:40]: Removed in the bug PR (by accident), so there we go! https://github.com/openshift/openshift-docs/pull/63057

File: modules/microshift-install-rpms.adoc [hunk 4]
```diff
@@ -31,7 +31,7 @@ $ sudo subscription-manager repos \
 $ sudo dnf install -y microshift
 ----
 
-. Optional: Install greenboot for {product-title} by running the following command:
+. Install greenboot for {product-title} by running the following command:
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-install-rpms.adoc:34]: @ggiguash Is this true for 4.13, or just 4.14?

**Comment by ShaunaDiaz** [comment]
@ggiguash This one should be lgtm? The bug PR is here: https://github.com/openshift/openshift-docs/pull/63057. That one will fix the RPMs names in both doc versions (4.13 and 4.14). Can you lgtm that one, too, if it looks good? Thank you!

**Comment by ggiguash** [comment]
This PR is LGTM for me.

File: modules/microshift-install-rpms.adoc [hunk 5]
```diff
@@ -31,13 +31,6 @@ $ sudo subscription-manager repos \
 $ sudo dnf install -y microshift
 ----
 
-. Optional: Install greenboot for {product-title} by running the following command:
```
Comment (dfitzmau) [inline modules/microshift-install-rpms.adoc:34]: Hi @ShaunaDiaz . This command was removed because Greenboot is now automatically installed on OCP?

Comment (ShaunaDiaz) [reply modules/microshift-install-rpms.adoc:34]: @dfitzmau Yes it is required, and no, this is for MicroShift. It has nothing to do with OCP.

Comment (dfitzmau) [reply modules/microshift-install-rpms.adoc:34]: OK. So Greenboot is now automatically installed on MicroShift?

Comment (dfitzmau) [reply modules/microshift-install-rpms.adoc:34]: Command removal is OK as now Greenboot is installed on MicroShift.

File: microshift_install/microshift-greenboot.adoc (hunk 3 above)
Comment (mburke5678) [inline microshift_install/microshift-greenboot.adoc:9]: Forgive me if this is a dumb question, should these mentions of `greenboot-default-health-check` be `greenboot-default-health-checks` based on https://github.com/openshift/openshift-docs/pull/63057/?

**Comment by mburke5678** [comment]
@ShaunaDiaz @dfitzmau One question. Otherwise LGTM

File: microshift_install/microshift-greenboot.adoc (hunk 3 above)
Comment (ShaunaDiaz) [reply microshift_install/microshift-greenboot.adoc:9]: @mburke5678 Yes, but I am not correcting it here also or it may cause the cherry pick to fail.

**Comment by ShaunaDiaz** [comment]
aa30633 rebased to pick up changes to RPM names from #63057

**Pull request merged**